src/App.tsx

```diff
@@ -1,8 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goodsFromServer = [
+type PreparedGoods = (a: string[], b: SortType, c: boolean) => string[];
+type Props = {
+  goods: string[];
+};
+
+enum SortType {
+  a = 'alphabet',
+  l = 'length',
+  default = '',
```

Comment:
The `default` value in the `SortType` enum is an empty string. Ensure that this case is handled properly in the `prepareGoods` function to avoid unexpected behavior.